### DISCUSSION TITLE
Hidden items

### DISCUSSION THREAD

**Discussion by gg-gg1** [deckerst/aves#171] [Ideas]
Hi there @deckerst . Thanks for the reply on my request. I was asking about the feature because I saw it working on simple gallery app from fdroid without asking me any permissions. I would like the feature to be implemented in the aves app so I don`t use 2 gallery apps, just the aves. It is not something very important, but it will be nice to have. 
P.S.: Sorry for the pull request. I am new on the github and learning how things works.

Thanks

**Comment by Donkey-Doug** [comment]
Related:

https://github.com/deckerst/aves/issues/84

**Comment by deckerst** [comment]
> working on simple gallery app from fdroid without asking me any permissions

Simply speaking, Simple Gallery has made a technical choice that allows them to ignore modern Android permission requirements.

Technically speaking, [Simple Gallery is targeting SDK 29 (Android 10)](https://github.com/SimpleMobileTools/Simple-Gallery/blob/master/app/build.gradle#L19), which means:
- they do not have to follow permission requirements that came with Android 11
- they cannot use features that came with Android 11 (or later)
- [since November 2021, they can no longer update their app on the Play Store](https://developer.android.com/google/play/requirements/target-sdk).

The last point may not impact you as you use F-Droid, but it is ethically interesting. Google's approach is to push the apps to follow modern requirements, that are deemed more secure and more respectful of user's privacy. I am not aware of F-Droid's view on the matter, but if they actually care about security and privacy, they should probably also enforce apps to target the latest Android SDK.

> Sorry for the pull request. I am new on the github and learning how things works.

No worries!
